app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -59,12 +74,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
         btnLogin.setOnClickListener(v -> {
 
-            String email = etEmail.getText().toString();
+            String email = etEmail.getEditText().getText().toString();
             String password = etPassword.getText().toString();
 
             presenter.login(email, password);
         });
 
+        savedEmails = new HashSet<>();
+        sharedPrefs = PreferenceManager.getDefaultSharedPreferences(this);
```

Comment:
We don't add logic in views. Please use UtilModel and LoginPresenter. Also write appropriate unit tests.

Thanks